frontend/src/components/intake/ShasIntakeForm.vue

```diff
@@ -333,16 +398,26 @@ onBeforeMount(async () => {
                       :bold="false"
                       :disabled="!editable"
                       :options="YesNo"
+                      @on-change="() => setFieldValue('basic.registeredName', null)"
+                    />
+                    <AutoComplete
+                      v-if="values.basic.isDevelopedInBC === BASIC_RESPONSES.YES"
+                      class="col-6 pl-0"
+                      name="basic.registeredName"
+                      :bold="false"
+                      :disabled="!editable"
+                      :editable="true"
+                      :force-selection="true"
+                      :get-option-label="getRegisteredNameLabel"
```

Comment:
Is this required? The function just returns the original object, which should be what is done automatically within PrimeVue. 

If required, is it possible to inline this, or a PrimeVue quirk requiring an actual function call? I can't recall. `:get-option-label="(e) => e"`